ext/action-popup.html

```diff
@@ -11,94 +11,64 @@
     <link rel="icon" type="image/png" href="/images/icon48.png" sizes="48x48">
     <link rel="icon" type="image/png" href="/images/icon64.png" sizes="64x64">
     <link rel="icon" type="image/png" href="/images/icon128.png" sizes="128x128">
+    <link rel="stylesheet" type="text/css" href="/css/material.css">
     <link rel="stylesheet" type="text/css" href="/css/action-popup.css">
     <script src="/js/pages/action-popup-main.js" type="module"></script>
 </head>
 <body>
-
-<div id="loading">
-    Loading...
-</div>
-
-<div id="action-popups">
-<div id="mini">
-    <label class="toggle">
-        <input type="checkbox" class="enable-search">
-        <div class="toggle-group">
-            <span class="toggle-on">On</span>
-            <span class="toggle-off">Off</span>
-            <span class="toggle-handle"></span>
-        </div>
-    </label>
-    <div class="nav-button-container">
-        <button type="button" class="nav-button action-select-profile" title="Change default profile" hidden>
-            <span class="icon" data-icon="profile"></span>
-            <span class="profile-select-container"><select class="profile-select">
-                <optgroup label="Default Profile" class="profile-select-option-group"></optgroup>
-            </select></span>
-        </button>
-        <a tabindex="0" class="nav-button action-open-settings" title="Settings" data-hotkey='["global:openSettingsPage","title","Settings ({0})"]'>
-            <span class="icon" data-icon="cog"></span>
-            <div class="nav-button-warning-badge no-dictionaries-enabled-warning" hidden>
-                <div class="nav-button-warning-badge-outer"></div>
-                <div class="nav-button-warning-badge-inner"></div>
+    <div>
+        <div tabindex="0" class="action-item action-select-profile" hidden>
+            <div class="action-item-inner">
+                <div class="action-item-left">
+                    <h2>Profile</h2>
+                </div>
+                <div class="action-item-right">
+                    <select class="profile-select" id="profile-select">
```

Comment:
This dropdown can only handle about 8 character strings before it cuts out and looks weird

<img width="191" alt="image" src="https://github.com/user-attachments/assets/4ba0d95b-d8eb-4a54-b3be-909f74476114">

Reply:
I added an ellipsis so it doesnt run over the dropdown icon